### PR TITLE
Fix track roomba

### PR DESCRIPTION
**Pull request by Quinnan-Gill**
I modified the abstract to test the new features with track_roomba

### CODE REVIEW

**Review by asaba96 — COMMENTED** [review]
Looks good, but PR in iarc7_motion needs merged first

File: scripts/track_roomba.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def track_roomba_land():
     roomba_id = roomba_id [0:len(roomba_id)-10]
 
     # Test tracking
-    goal = QuadMoveGoal(movement_type="track_roomba", frame_id = roomba_id, tracking_mode = True)
+    goal = QuadMoveGoal(movement_type="track_roomba", frame_id = roomba_id, time_to_track = 5.0, overshoot = .5)
```
Comment (amiller27) [inline scripts/track_roomba.py]: In `iarc7_abstract`, the python convention is no spaces around `=` in named function args

Comment (aftersomemath) [inline scripts/track_roomba.py]: Check line length?

**Review by aftersomemath — DISMISSED** [review]

**Comment by aftersomemath** [comment]
Note to myself that this needs to be updated to reflect the accompanying changes in the PR in HLM

**Review by amiller27 — CHANGES_REQUESTED** [review]
Looks good.

One thing, the python convention for named parameters is no spaces around the `=` sign, can you change that where you're making the `QuadMoveGoal` objects?

Also, has the RoombaRequestExecuter been tested in the sim?

**Comment by Quinnan-Gill** [comment]
Fixed the conventions

**Review by amiller27 — APPROVED** [review]

**Pull request merged**